scripts/release_info.py

```diff
@@ -51,16 +56,29 @@
                     'CURRENT_COLLECTION_SCHEMA_VERSION']
 FIRST_OPPIA_COMMIT = '6a7138f5f603375e58d1dc3e1c4f1c80a126e249'
 NO_LABEL_CHANGELOG_CATEGORY = 'Uncategorized'
+# This should match the indentation of span and li elements used
```

Comment:
I think you could have the script test/assert this? Better than just leaving a comment.

Could do it in a backend test (scripts/release_info_test.py is fine) so that if someone changes the directive and indents everything 2 spaces to the right (say), then that will be caught and these constants will be fixed.

Reply:
Instead of adding a test, should I do this: select the line which contains `<span>A</span>` and then find the indentation using this line (same for li element) since a test will also need to select this line.

Reply:
Yes, that's fine too. And throw an error if there's no such line. (In general be careful about handling all the possible error cases and being noisy about them, since there's no guarantee that the files you're working with won't change in the future.)

Reply:
Done.